api/src/main/java/org/pmiops/workbench/auth/ProfileService.java

```diff
@@ -88,6 +102,10 @@ public Profile getProfile(User user) {
     if (user.getAuthorities() != null) {
       profile.setAuthorities(new ArrayList<>(user.getAuthorities()));
     }
+    if (userWithPageVisits.getPageVisits() != null) {
```

Comment:
would it be wise to add a not-empty check? Saves a call to stream over an empty set.

Reply:
I thought that if the array was empty, no action would be taken?

Reply:
That is mostly true, i.e., nothing will be collected in the collector. In the meantime, however, you're calling stream on it, which would be unnecessary.